.bazelversion

```diff
@@ -0,0 +1 @@
+last_downstream_green
```

Comment:
will this mean all our tests will run with the last_downstream_green? Will this mean we get the latest features (and breaking features)? Not sure we want to commit to that at the top level directory. if you want this for your tests you could add a sub project (e.g., https://github.com/bazelbuild/bazel-toolchains/tree/master/tests/rbe_repo/examples/bazel_toolchains_client and add its top level dir to https://github.com/bazelbuild/bazel-toolchains/blob/master/.bazelignore.

Reply:
(sorry approved by mistake, lets figure out if we can remove this file from this PR or move it to a sub project)

Reply:
My bad. I did not mean to add this to the PR.
I will remove it.